client/post-editor/controller.js

```diff
@@ -253,4 +253,14 @@ export default {
 		page.redirect( redirectWithParams );
 		return false;
 	},
+
+	gutenberg: async function( context, next ) {
+		const siteId = await getSelectedSiteId( context.store.getState() );
+		const editor = await getSelectedEditor( siteId );
```

Comment:
do you need a try / finally to wrap these? If they reject and throw, you'll never call next and end up showing nothing.